docs/getting-started-installation.md

```diff
@@ -56,7 +55,7 @@ root-directory
     └── static
 ```
 
-## Running the example website
+## Running the Example Website
```

Comment:
The casing is intentional and consistent across the whole docs. There's no standard convention for headings and we chose to use a "Foo bar qux" over "Foo Bar Qux" convention. Please revert them.